cmd/notation/verify.go

```diff
@@ -90,15 +87,18 @@ Example - [Experimental] Verify a signature on an OCI artifact identified by a t
 			if opts.maxSignatureAttempts <= 0 {
 				return fmt.Errorf("max-signatures value %d must be a positive number", opts.maxSignatureAttempts)
 			}
+			if cmd.Flags().Changed("allow-referrers-api") {
+				fmt.Fprintln(os.Stderr, "Warning: flag '--allow-referrers-api' is deprecated and ignored.")
```

Comment:
Please update message to show it would be removed in future versions like major version bump.